ajax/libs/jsfx/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "jsfx",
+  "filename": "jsfx.min.js",
+  "version": "1.1.0",
+  "description": "Package for sound effect generation.",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/loov/jsfx.git"
+  },
+  "keywords": [
+    "audio",
+    "sound",
+    "effects",
+    "jsfx",
+    "gamedev"
+  ],
+  "author": "loov",
+  "license": "MIT",
+  "homepage": "http://loov.io/jsfx",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/loov/jsfx.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "jsfx.js",
+          "jsfx.min.js"
```

Comment:
It's more flexible to use `jsfx*` instead of `jsfx.js` and `jsfx.min.js` for future maintainance.